vllm/model_executor/model_loader/loader.py

```diff
@@ -98,8 +98,8 @@ def _get_quantization_config(
     """Get the quantization config."""
     if model_config.quantization is not None:
         quant_config = get_quant_config(model_config, load_config)
-        capability_tuple = current_platform.get_device_capability()
-
+        capability_tuple = current_platform.get_device_capability() \
+                            if current_platform.is_cuda_alike() else None
```

Comment:
Please revert this as we decided to add quantization in the next PR.

Reply:
removed